src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPlugin.kt

```diff
@@ -26,7 +28,20 @@ open class MavenPublishPlugin : Plugin<Project> {
 
     val pom = MavenPublishPom.fromProject(p)
     p.setCoordinates(pom)
-    p.configureTargets(extension)
+    p.checkProperties()
+    p.configureArchivesTasks()
+
+    p.publishing.repositories.maven { repo ->
+      repo.name = "mavenCentral"
```

Comment:
Nitpicking: the repository itself is OSS staging. But everyone knows about MavenCentral so I guess it makes sense. I'm not sure which one is best.

Reply:
I went with that for easier understandability. When using it, that's where you want your artifacts to end up with and most probably don't know all the details https://twitter.com/saketme/status/1357436828861759491